### PR TITLE
[class] Replace 'could' and 'might'

### PR DESCRIPTION
**Pull request by jensmaurer**
as directed by ISO/CS.

Partially addresses #4319

### CODE REVIEW

File: source/classes.tex [hunk 1]
```diff
@@ -614,7 +614,7 @@
 In a \grammarterm{member-declarator} for a bit-field,
 the \grammarterm{constant-expression} is parsed as
 the longest sequence of tokens
-that could syntactically form a \grammarterm{constant-expression}.
+that would syntactically form a \grammarterm{constant-expression}.
```
Comment (zygoloid) [inline source/classes.tex:617]: I think we can avoid the hypothetical here (and in other cases where we describe tentative parses):
```suggestion
that matches the syntax of a \grammarterm{constant-expression}.
```

File: source/classes.tex [hunk 2]
```diff
@@ -2303,9 +2303,9 @@
 \begin{note}
 A
 \tcode{return}
-statement\iref{stmt.return} in a destructor might not directly return to the
+statement\iref{stmt.return} in a destructor does not directly return to the
 caller; before transferring control to the caller, the destructors for the
-members and bases are called.
+members and bases (if any) are called.
```
Comment (zygoloid) [inline source/classes.tex:2308]: The pre-existing wording is unnecessarily verbose. I think the "does not" might be confusing in the case where there are no members or bases. How about replacing this with:

"A `return` statement ([stmt.return]) in a destructor calls the destructors for the members and bases (if any) before transferring control to the caller."

File: source/classes.tex [hunk 3]
```diff
@@ -2692,7 +2692,7 @@
 in a
 \grammarterm{conversion-function-id}
 is the longest sequence of
-tokens that could possibly form a \grammarterm{conversion-type-id}.
+tokens that would possibly form a \grammarterm{conversion-type-id}.
```
Comment (zygoloid) [inline source/classes.tex:2695]:
```suggestion
tokens that match the syntax of a \grammarterm{conversion-type-id}.
```

File: source/classes.tex [hunk 4]
```diff
@@ -6546,7 +6546,7 @@
 requiring a constant expression\iref{expr.const}
 and in constant initialization\iref{basic.start.static}.
 \begin{note}
-Copy elision might be performed
+It is possible for copy elision to be performed
```
Comment (zygoloid) [inline source/classes.tex:6549]:
```suggestion
It is possible that copy elision is performed
```

File: source/classes.tex (hunk 1 above)
Comment (jensmaurer) [reply source/classes.tex:617]: fixed

File: source/classes.tex (hunk 2 above)
Comment (jensmaurer) [reply source/classes.tex:2308]: fixed

File: source/classes.tex (hunk 3 above)
Comment (jensmaurer) [reply source/classes.tex:2695]: "that matches" (singular); it's referring to the sequence

fixed

**Comment by jensmaurer** [comment]
applied, squashed, and force-pushed

**Pull request merged**